### PR TITLE
Add Open Liberty APIs/SPIs to the SEO on docs

### LINKED ISSUE #3293: api/spi: add content description
api/spi spo: heads-meta.hbs to add content (description)? for Open Liberty API/SPI

### PR DESCRIPTION
**Pull request by steven1046**
## What was changed and why?
Added Open Liberty APIs/SPIs to the SEO for docs.
Fixes #3293

## Tested using browser:
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)

### CODE REVIEW

**Comment by steven1046** [comment]
@chirp1, can you please review the wording change for SEO to include Open Liberty APIs/SPIs? Since Open Liberty are the only SPIs on the website, i added API to each of the other kind of javadocs in the sentence to show only Open Liberty have APIs and SPIs. Maybe we could make it a new sentence if it's too long.

Old sentence:
`Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE, MicroProfile and Java EE APIs.`

New sentence:
`Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs/SPIs.`

**Comment by chirp1** [comment]
@dmuelle @ramkumar-k-9286 Hi! Since you are working on the API/SPI landing pages, would you review for Steven? Thanks!

**Comment by dmuelle** [comment]
Hi Steven- where does this text show up?

File: src/main/content/antora_ui/src/partials/head-meta.hbs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {{else if page.attributes.description }}
 <meta name="description" content="{{page.attributes.description}}">
 {{ else }}
-<meta name="description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE, MicroProfile and Java EE APIs.">
+<meta name="description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs/SPIs.">
```
Comment (dmuelle) [inline src/main/content/antora_ui/src/partials/head-meta.hbs]:
```suggestion
<meta name="description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds by using open cloud-native Java. This content covers Open Liberty basics, development, security, deployment, and operations topics. It also includes  Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs and SPIs.">
```

File: src/main/content/antora_ui/src/partials/head-meta.hbs [hunk 2]
```diff
@@ -23,7 +23,7 @@
 {{else if page.attributes.description }}
 <meta name="twitter:description" content="{{ page.attributes.description }}" />
 {{ else }}
-<meta name="twitter:description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE, MicroProfile and Java EE APIs." />
+<meta name="twitter:description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs/SPIs." />
```
Comment (dmuelle) [inline src/main/content/antora_ui/src/partials/head-meta.hbs]:
```suggestion
<meta name="description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds by using open cloud-native Java. This content covers Open Liberty basics, development, security, deployment, and operations topics. It also includes  Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs and SPIs.">
```

File: src/main/content/antora_ui/src/partials/head-meta.hbs [hunk 3]
```diff
@@ -39,7 +39,7 @@
 {{else if page.attributes.description }}
 <meta property="og:description" content="{{ page.attributes.description }}" />
 {{ else }}
-<meta property="og:description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE, MicroProfile and Java EE APIs." />
+<meta property="og:description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds using open cloud-native Java. Covers Open Liberty basics, development, security, deployment, and operations topics, as well as Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs/SPIs." />
```
Comment (dmuelle) [inline src/main/content/antora_ui/src/partials/head-meta.hbs]:
```suggestion
<meta property="og:description" content="Open Liberty documentation and reference materials for developers to build applications and for administrators and operation teams to manage DevOps and deploy workloads to clouds by using open cloud-native Java. This content covers Open Liberty basics, development, security, deployment, and operations topics. It also includes  Javadoc for Jakarta EE APIs, MicroProfile APIs, Java EE APIs, and Open Liberty APIs and SPIs.">
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review- minor edits for clarity and sentence length

**Comment by steven1046** [comment]
> Hi Steven- where does this text show up?

When people see links to our docs on Google and Twitter.

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Pull request merged**